modules/spring-config/src/main/java/org/mule/runtime/config/spring/MuleDocumentLoader.java

```diff
@@ -56,7 +60,26 @@ public void handle(short operation, String key, Object data, Node src, Node dst)
         }
     };
 
-    private final DocumentLoader defaultLoader = new DefaultDocumentLoader();
+    private final DocumentLoader defaultLoader = new DefaultDocumentLoader()
+    {
+        @Override
+        protected DocumentBuilderFactory createDocumentBuilderFactory(int validationMode, boolean namespaceAware)
+                throws ParserConfigurationException
+        {
+            DocumentBuilderFactory factory = super.createDocumentBuilderFactory(validationMode, namespaceAware);
+            try
+            {
+                factory.setFeature(DEFER_NODE_EXPANSION_FEATURE_KEY, false);
```

Comment:
we should run a profile on this before introducing it. For example, take  the most heavy metal templates and try it with thouse